modules/core/src/lib/layer.ts

```diff
@@ -433,12 +433,11 @@ export default abstract class Layer<PropsT extends {} = {}> extends Component<
   }
 
   // Default implementation
-  // Sublayers chould override this method to provide an accurate calculation of the bounds
-  getBounds(): [number[], number[]] | null {
-    const attributeManager = this.getAttributeManager();
-    if (!attributeManager) return null;
-    const {positions, instancePositions} = attributeManager.attributes;
-    return (positions || instancePositions)?.getBounds();
+  // Sublayers should override this method to provide an accurate calculation of the bounds
+  getBounds(attributeNames?: string[]): [number[], number[]] | null {
```

Comment:
I would avoid all the `super.getBounds()` calls. At this point, it's easy enough and clearer to just call the `AttributeManager` method. Also `layer.getBounds()` is a public method so this new argument will probably confusing to the user.

```js
// Layer
getBounds() {
  this.getAttributeManager()?.getBounds(['positions', 'instancePositions']);
}

// LineLayer
getBounds() {
  this.getAttributeManager()?.getBounds(['instanceSourcePositions', 'instanceTargetPositions']);
}
```